scripts/extract-package-code.R

```diff
@@ -1,3 +1,17 @@
+library(fs)
+library(optparse)
+library(purrr)
+library(stringr)
```

Comment:
Would it make sense to make a dependency to genthat - not that I would like to promote it in any way, but simply to prevent code duplication.

Reply:
It does not sound very reliable to have one research project depend on another for ~150 lines of code which are not even the main purpose of `genthat`. 

Extracting vignettes, examples and tests is something common to all tracing work and the code to do that is small enough that we can avoid having our tracing repos depend mutually for it.

Having separate copies also lets us evolve them independently based on the needs of the project. If we use genthat for this and tomorrow this API is updated in genthat, we will have to update this repo as well. We already depend on `evaldyntracer` and `R-dyntrace` for this repo.

Reply:
My idea was to use a particular tag of genthat so in that case it will always work. But if this makes it easier, I'm all for it.